src/main/java/com/epam/aidial/core/config/Deployment.java

```diff
@@ -22,4 +23,11 @@ public abstract class Deployment {
     private Features features;
     private List<String> inputAttachmentTypes;
     private Integer maxInputAttachments;
+    /**
+     * Default parameters are applied if a request doesn't contain them in OpenAI chat/completions API call.
+     * <p>
+     *     Note. Parameters' values are converted to strings, e.g. 0.25 -> "0.25" or true -> "true"
+     * </p>
+     */
+    private Map<String, String> defaults = Map.of();
```

Comment:
We should generalize from `String` to an arbitrary atomic Object - strings, floats, ints.

Reply:
@adubovik is that crucial to map exactly data type, eg float -> float?
the code does float to string and the parameter always comes to the deployment as a string value.

Reply:
fixed